clients/batch.go

```diff
@@ -44,6 +44,25 @@ func (tb *TransferBatch) String() string {
 	return str
 }
 
+// Json will convert the transfer batch to a string
+func (tb *TransferBatch) Json() string {
```

Comment:
please remove this, add json string tags for the TransferBatch & DepositTransfer structs
If needed, you can implement the Marshaler interface from the standard `json` package.

Reply:
removed